Podfile

```diff
@@ -10,9 +10,12 @@ plugin 'cocoapods-keys', {
     'HockeyBetaSecret',
     'MixpanelProductionAPIClientKey',
     'MixpanelStagingAPIClientKey',
-    'CardflightAPIClientKey',
-    'CardflightMerchantAccountToken',
-    'StripePublishableKey'
+    'CardflightProductionAPIClientKey',
+    'CardflightProductionMerchantAccountToken',
+    'StripeProductionPublishableKey',
+    'CardflightStagingAPIClientKey',
+    'CardflightStagingMerchantAccountToken',
+    'StripeStagingPublishableKey'
```

Comment:
We don't have a Makefile; both OSS users and CI use this lane, so we should be good.